docs/installation.md

```diff
@@ -29,9 +33,9 @@ Change to the a directory where you keep your projects, and create a new SMK app
 You will be asked some questions about your new application.
 Once they are answered you will have a new skeleton application at `projects/my-new-app`.
 
-You will also be asked whether you want to run the app. You can choose yes to launch the app immediately. 
+You will also be asked whether you want to edit the app. Choosing yes will launch the interactive editor.
```

Comment:
Thanks, this fixes my recent changes - I should have actually run 'smk create' instead of going from memory :(

Reply:
One other thing - "interactive editor" - this launches the web GUI, and maybe that's what you meant, but the CLI wizard is also arguably an interactive editor, so consider making this more specific.

Reply:
Changed to 'web GUI editor'